LCD Interfacing/LCD.c

```diff
@@ -0,0 +1,95 @@
+#include "LCD.h"
+
+void lcd_send_nibble(uint8_t ui8RS, uint8_t ui8Val)
+{
+	/* is the value cmd or data*/
+	if( ui8RS == 0 )

```

Comment:
Use macro for RS value.
Eg: if (u8RS == RS_COMMAND_D)   -> Imporves readability

Reply:
Done